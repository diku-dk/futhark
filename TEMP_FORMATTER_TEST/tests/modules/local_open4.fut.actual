fb8204dc015611cf71549ce27ca92afc
