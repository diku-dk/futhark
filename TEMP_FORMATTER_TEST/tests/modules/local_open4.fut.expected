e373f0516f9f3c68e1690c4a07acc941
